ssapy/body.py

```diff
@@ -103,7 +107,7 @@ def __init__(self):
         from jplephem.spk import SPK
         from . import datadir
 
-        fn = os.path.join(datadir, "de430.bsp")  # https://naif.jpl.nasa.gov/pub/naif/LUCY/kernels/spk/de430s.bsp.lbl
+        fn = os.path.join(datadir, "de430.bsp")  # https://naif.jpl.nasa.gov/pub/naif/LUCY/kernels/spk/de430s.bsp
```

Comment:
The `.lbl` part of the URL takes you to the fild documentation, which I think was the point. Please return the url to what it was or list both the source URL and the documentation URL.

Reply:
Fixed